site/docs/.vuepress/config.ts

```diff
@@ -182,15 +182,15 @@ export default defineUserConfig<DefaultThemeOptions>({
                     link: "/plugins/hydrate.md",
                   },
                   {
-                    text: "Retry API requests (auto-retry)",
+                    text: "Retry API Requests (auto-retry)",
                     link: "/plugins/auto-retry.md",
                   },
                   {
-                    text: "Flood control (transformer-throttler)",
+                    text: "Flood Control (transformer-throttler)",
                     link: "/plugins/transformer-throttler.md",
                   },
                   {
-                    text: "Rate limit users (ratelimiter)",
+                    text: "Rate Limit Users (ratelimiter)",
```

Comment:
```suggestion
                    text: "Rate Limit Users (rate-limiter)",
```
or:
```suggestion
                    text: "Rate Limit Users (rate limiter)",
```

Reply:
The word in brackets is the plugin name. We cannot change it without publishing new packages for Node and Deno.